src/App.tsx

```diff
@@ -15,48 +18,113 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabetical = 'Alphabetical',
+  Length = 'Length',
+}
+
+const sortGoods = (goodsArray: string[], sortType: SortType,
+  isReverse: boolean): string[] => {
+  let sortedGoods = [...goodsArray];
+
+  switch (sortType) {
+    case SortType.Alphabetical:
+      sortedGoods = sortedGoods.sort();
+      if (isReverse) {
+        sortedGoods.reverse();
+      }
+
+      return sortedGoods;
+    default:
+      sortedGoods = [...goodsFromServer].sort((a, b) => {
```

Comment:
- by default it should return just [...goodsFromServer], because it's a default case 
- this sorting logic move to the separate case (Length)